.gitignore

```diff
@@ -56,3 +56,5 @@ deploy_key
 
 # Custom directories
 dist/
+
+cucumber.js
```

Comment:
I think that's not required anymore, right?

Reply:
Running the ITs would generate a mock cucumber report named cucumber.js which I think should no go to the repository.
I will rename the report to make it more obvious.

Reply:
Got it. Then maybe use a sub-folder "reports" and exclude the folder here (like `dist/`).

Reply:
Good Idea, will do so